libs/partners/google-genai/langchain_google_genai/chat_models.py

```diff
@@ -509,6 +521,7 @@ def validate_environment(cls, values: Dict) -> Dict:
         values["client"] = genai.GenerativeModel(model_name=model)
```

Comment:
I think the place to use the model instance (i.e. self) safety_settings is here when the client is created:
safety_settings = values.get("safety_settings")  # I assume missing key is None value
genai.GenerativeModel(model_name=model, safety_settings=safety_settings)